internal/cephes/cephes.go

```diff
@@ -21,7 +21,8 @@ var (
 )
 
 const (
-	machEp = 1.11022302462515654042e-16 // 2^-53
-	maxLog = 7.09782712893383996732e2   // log(2^127)
-	minLog = -7.451332191019412076235e2 // log(2^-128)
+	machEp  = 1.11022302462515654042e-16 // 2^-53
```

Comment:
Interestingly, the comments for `maxLog` and `minLog` are misleading. `7.09782712893383996732e2` is actually `log(2^1024)` while `-7.451332191019412076235e2` is `log(2^-1075)`. All three of those constants (`machEp`, `maxLog` and `minLog`) predate this PR. Do you want me to leave them be so that can be addressed in a separate PR?